docs/providers/oci.md

```
@@ -68,6 +68,8 @@ authoring-time gaps that don't survive into the manifest.
 | [OCI-004](#oci-004) | Image layer references an arbitrary URL (foreign layer) | <span class="pg-sev pg-sev--high">HIGH</span> |  |
 | [OCI-005](#oci-005) | Image manifest is missing the ``image.licenses`` annotation | <span class="pg-sev pg-sev--low">LOW</span> |  |
 | [OCI-006](#oci-006) | Image has an excessive layer count | <span class="pg-sev pg-sev--low">LOW</span> |  |
+| [OCI-007](#oci-007) | Image manifest uses legacy schemaVersion 1 (no content addressing) | <span class="pg-sev pg-sev--high">HIGH</span> |  |
+| [OCI-008](#oci-008) | Manifest references digest using non-sha256 hash | <span class="pg-sev pg-sev--high">HIGH</span> |  |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)